docs/layout/drawer/index.hbs

```diff
@@ -0,0 +1,167 @@
+---
+title: Drawer component
+description: Add a fixed drawer-like element for your mobile or desktop navigation.
+nav: side/layout/drawer
+order: 2
+lunr: true
+toc:
+  - title: Overview
+    slug: overview
+  - title: Top drawer
+    slug: top-drawer
+  - title: Side drawer
+    slug: side-drawer
+  - title: Persistent side drawer
+    slug: persistent-side-drawer
+---
+
+{{#> page}}
+  {{#*inline "content"}}
+
+    <h1 class="docs-h2">{{title}}</h1>
+
+    <p class="lead">The drawer is a flexible layout component. It helps you
+    create togglable mobile menus. On larger viewports,
+    it can also stick persistently on one side.
+    This website you're on even uses both depending on the viewport!</p>
+
+    {{> table-of-contents data=toc}}
+
+    <h2 id="overview" class="docs-h3">Overview</h2>
+
+    <p>Most probably, you'll use the drawer as a mobile menu.
+    The recommended way is to let it slide in from the top of the screen,
+    where it then presents a menu.
+    However, the drawer also supports toggling
+    from both sides of the screen.</p>
+
+    <p>These are the <em>interactive</em> ways of using the drawer:</p>
+
+    <ul>
+      <li>Toggle from top</li>
+      <li>Toggle from left side</li>
+      <li>Toggle from right side</li>
+    </ul>
+
+    <p>It is also possible to use a static approach and make the drawer
+    persist on either the left or the right side of the screen. This
+    can be helpful for websites that are solely driven by a navigation
+    and can afford the always occupied screen estate:</p>
+
+    <ul>
+      <li>Persist on left side</li>
+      <li>Persist on right side</li>
+    </ul>
+
+    <p>Ultimately, all these ways of using the drawer can also be combined.
+    For example, you can make the drawer togglable on small devices
+    and persistent on large screens.</p>
+
+    <h2 id="top-drawer" class="docs-h3">Top drawer</h2>
```

Comment:
I'm just a guy that want's to see code first usually and I think most other devs are like that too.
Also just reading about a bunch of properties in a prose text is kind of confusing, whereas seeing a structure and then having the possibility to make the association between your reference to a property or element in the text and the property in the context of the code helps understand it.